.github/workflows/bazel.yaml

```diff
@@ -118,7 +118,16 @@ jobs:
       - uses: bazel-contrib/setup-bazel@0.8.0
         with:
           repository-cache: ${{ inputs.mount_bazel_caches }}
-          bazelrc: common --announce_rc --color=yes --enable_bzlmod=${{ matrix.bzlmodEnabled }} --enable_workspace=${{ ! matrix.bzlmodEnabled }}
+          bazelrc: |
+            common --announce_rc
+            common --color=yes
+            common --enable_bzlmod=${{ matrix.bzlmodEnabled }}
+
+      - name: Configure bazelrc --enable_workspace (bazelversion >= 7.1.0)
+        working-directory: ${{ matrix.folder }}
+        if: ${{ !startsWith(matrix.bazelversion, '6.') && !startsWith(matrix.bazelversion, '7.0.') }}
+        run: |
+          echo "common --enable_workspace=${{ ! matrix.bzlmodEnabled }}" >> .bazelrc
```

Comment:
nit: now that we only write the one line, I think it would be better to have the `!matrix.bzlmodEnabled` condition in a clause of the the dedicated `if` expression above, then this is just `common --enable_workspace`

Reply:
i think i just applied what you suggested :)